website/content/docs/agent/options.mdx

```diff
@@ -720,7 +743,7 @@ Valid time units are 'ns', 'us' (or 'µs'), 'ms', 's', 'm', 'h'."
   not impacted.
 
 - `acl_agent_master_token` ((#acl_agent_master_token_legacy)) - **Deprecated
-  in Consul 1.4.0. See the [`acl.tokens.agent_master`](#acl_tokens_agent_master)
+  in Consul 1.4.0. See the [`acl.tokens.agent_recovery`](#acl_tokens_agent_recovery)
```

Comment:
If someone has this in their config file, they probably aren't upgrading directly from pre 1.4 to 1.11+. If they are upgrading to something <1.11, this flag doesn't exist, and the docs for it don't mention the older name of the flag. We need to fix that somehow... perhaps by referencing the old name from the new name?

Reply:
Good point! Let's go with the old name.